Jenkinsfile

```diff
@@ -89,7 +90,12 @@ def ZOWE_LICENSE_ZIP_URL = "https://zowe.jfrog.io/zowe/$ARTIFACTORY_RELEASE_REPO
 /**
 * Master branch
 */
-def MASTER_BRANCH = "master"
+def MASTER_BRANCH = "publish-next"
+
+/**
+* If true, no artifacts will be published even when branch is MASTER_BRANCH
+*/
+def DRY_RUN = true

```

Comment:
Nice implementation of a DryRun.
Should it also be a parameter of the pipeline?

Reply:
If the pipeline already had other parameters, I would probably have added DryRun as a new one. If you think it's worthwhile to create a parameter list with DryRun as the first one, I don't mind doing so. 🙂

Reply:
no need to worry about this now. maybe in the future we can add a parameter list 😋